.dockerignore

```
@@ -12,5 +12,6 @@ back/log/*
 back/tmp/*
 back/public/uploads
 back/secrets/*
-back/!/secrets/*.example
+back/!secrets/*.example
```

@kogre does this seem right to you? It works when testing locally, but as you can see the README file came up with changes.

---

I think it needs to be rather this. The exclamation point signals an exception to the rules, and it needs to come at the start of the line. [Docs](https://docs.docker.com/engine/reference/builder/#dockerignore-file).
```suggestion
!back/secrets/*.example
```